routers/web/repo/repo.go

```diff
@@ -378,10 +378,6 @@ func RedirectDownload(ctx *context.Context) {
 	curRepo := ctx.Repo.Repository
 	releases, err := repo_model.GetReleasesByRepoIDAndNames(ctx, curRepo.ID, tagNames)
 	if err != nil {
-		if repo_model.IsErrAttachmentNotExist(err) {
```

Comment:
A hint for reviewers: `GetReleasesByRepoIDAndNames` only does simple database query, it never returns `ErrAttachmentNotExist`